packages/e2e-tests/register/seedPhrase.spec.js

```diff
@@ -0,0 +1,125 @@
+const { CreateAccountPage } = require("./models/CreateAccount");
+const { SetRecoveryOptionPage } = require("./models/SetRecoveryOption");
+const { SetupSeedPhrasePage } = require("./models/SetupSeedPhrase");
+const { VerifySeedPhrasePage } = require("./models/VerifySeedPhrase");
+
+const { describe, afterAll } = test;
+
+describe("Account Registration Using Seed Phrase", () => {
+    const testAccountId = generateTestAccountId();
+    let testAccountInstance;
+
+    afterAll(async () => {
+        testAccountInstance &&
+            (await testAccountInstance.deleteAccount("testnet"));
+    });
+
+    test("navigates to set account recovery page successfuly", async ({
+        page,
+    }) => {
+        const homePage = new HomePage(page);
+        await homePage.navigate();
+
+        await homePage.clickCreateAccount();
+        await expect(page).toMatchURL(/\/create$/);
+
+        const createAccountPage = new CreateAccountPage(page);
+        await createAccountPage.acceptTerms();
+        await createAccountPage.submitAccountId(testAccountId);
+        await expect(page).toMatchURL(
+            new RegExp(`/set-recovery/${testAccountId}`)
+        );
+    });
+    test("is able to select other recovery methods and navigate to phrase setup", async ({
+        page,
+    }) => {
+        const setRecoveryOptionPage = new SetRecoveryOptionPage(page);
+        await setRecoveryOptionPage.navigate(testAccountId);
+
+        await setRecoveryOptionPage.clickLedgerRecoveryOption();
+        await expect(page).toMatchAttribute(
+            setRecoveryOptionPage.getLedgerSelector(),
+            "class",
+            /active/
+        );
+
+        await setRecoveryOptionPage.clickEmailRecoveryOption();
+        await expect(page).toMatchAttribute(
+            setRecoveryOptionPage.getEmailSelector(),
+            "class",
+            /active/
+        );
+
+        await setRecoveryOptionPage.clickPhoneRecoveryOption();
+        await expect(page).toMatchAttribute(
+            setRecoveryOptionPage.getPhoneSelector(),
+            "class",
+            /active/
+        );
+
+        await setRecoveryOptionPage.clickSeedPhraseRecoveryOption();
+        await expect(page).toMatchAttribute(
+            setRecoveryOptionPage.getSeedPhraseSelector(),
+            "class",
+            /active/
+        );
+
+        await setRecoveryOptionPage.clickSeedPhraseRecoveryOption();
+        await setRecoveryOptionPage.submitRecoveryOption();
+
+        await expect(page).toMatchURL(
+            new RegExp(`/setup-seed-phrase/${testAccountId}/phrase`)
+        );
+    });
+    test("is able to verify seed phrase and access wallet", async ({
+        page,
+        context,
+    }) => {
+        // skip test on browsers that don't support clipboard API
+        await context
+            .grantPermissions(["clipboard-read", "clipboard-write"])
+            .catch(test.skip);
+        // skip test on mainnet
```

Comment:
Would be good for this comment to indicate what conditions need to be met for us to enable this on mainnet